Libbna/CUSTOM-CMS
# #203 default logo image not present

Testing steps: 
- Update database with new sql file
- Check your assets/images/logo folder contain the 'logo.png' file.
 else
- Confirm your database config table have only one entry and it has logo value 'assets/images/logo/logo.png' 
- And also confirm your assets/images/logo/logo.png file
